source/js/components/expander.js

```diff
@@ -20,17 +20,23 @@ var Panel = React.createClass({
     this.props.activateKey(itemKey);
   },
   render: function() {
+    var expanded = this.props.itemKey === this.props.activeKey;
+    var style = {
+      height: "0"
+    };
     var itemClassName = classnames("expander-item", {
-      "expander-item-active": this.props.itemKey === this.props.activeKey
-    });
-    var contentClassName = classnames("expander-content", {
-      "expander-content-active": this.props.itemKey === this.props.activeKey
+      "expander-item-active": expanded
     });
+    if (this.content && expanded) {
```

Comment:
from line 37-39
```jsx
<div ref={(element) => {this.content = element;}}>
  {this.props.children}
</div>
```
`this.content` is never `null`.  It would be either a `<div>` with or without children nodes?